indexers/dbms/PostgreSQLIndexer/__init__.py

```diff
@@ -0,0 +1,107 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+from typing import Optional
+
+
+from jina.executors.indexers import BaseIndexer
+from ..PostgreSQLIndexer.postgreshandler import PostgreSQLDBMSHandler
+
+
+class PostgreSQLDBMSIndexer(BaseIndexer):
+    # TODO:  this class need to be a subclass from the DBMSIndexer (when it's merged into master)
+    """:class:`PostgreSQLDBMSIndexer` PostgreSQL based KV Indexer.
```

Comment:
It's not a KV Indexer, it's a DBMS indexer (it stores a mapping of id --> vector; id --> metadata)

Reply:
Yes, typo from other classes